### ISSUE TITLE
add post filter for empty carmen:text

### ISSUE THREAD

**Issue by sbma44** [ingalls/pt2itp#249]
I suspect this is my fault, stemming from the titleCase code path where both the main and alt text are empty. We need to suppress the feature, not just return an empty string; otherwise the indexer fails.

**Comment by KaiBot3000** [comment]
Oh, cool, I wonder if this is what I'm seeing. I have a few states where the indexer seems to hang, and that I haven't debugged yet.

**Comment by sbma44** [comment]
implemented in #252, will be merged with https://github.com/ingalls/pt2itp/pull/248

**Comment by sbma44** [comment]
@KaiBot3000 fwiw when I've run to this the indexer has failed noisily, not just hung there, so I'm betting your issue might be something different
